src/cascadia/TerminalApp/App.cpp

```diff
@@ -176,6 +237,79 @@ namespace winrt::TerminalApp::implementation
         _ShowDialog(winrt::box_value(title), winrt::box_value(message), buttonText);
     }
 
+    // Method Description:
+    // - Displays a dialog for errors found while loading or validating the
+    //   settings. Uses the resources under the provided  title and content keys
+    //   as the title and first content of the dialog, then also displays a
+    //   message for whatever exception was found while validating the settings.
+    // - Only one dialog can be visible at a time. If another dialog is visible
+    //   when this is called, nothing happens. See _ShowDialog for details
+    // Arguments:
+    // - titleKey: The key to use to lookup the title text from our resources.
+    // - contentKey: The key to use to lookup the content text from our resources.
+    void App::_ShowLoadErrorsDialog(const winrt::hstring& titleKey,
+                                    const winrt::hstring& contentKey)
+    {
+        auto title = _resourceLoader.GetLocalizedString(titleKey);
+        auto buttonText = _resourceLoader.GetLocalizedString(L"Ok");
+
+        Controls::TextBlock warningsTextBlock;
+        // Make sure you can copy-paste
+        warningsTextBlock.IsTextSelectionEnabled(true);
+        // Make sure the lines of text wrap
+        warningsTextBlock.TextWrapping(TextWrapping::Wrap);

```

Comment:
Can we make these be pretty colors? Errors in red (or reddish) and warnings in yellow (or yellowish)?

Reply:
There is a ThemeResource `SystemControlErrorTextForegroundBrush`, which is red on light theme and yellow on dark theme o_O